Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/allowlist/internal/DccTicketingAllowListException.kt

```diff
@@ -21,12 +21,13 @@ class DccTicketingAllowListException(
         CLIENT_ERROR("Update of Allow list failed with client error."),
 
         ALLOWLIST_NO_MATCH("AllowList no match."),
+        SP_ALLOWLIST_NO_MATCH("Service Provider allowlist mismatch "),
     }
 
     override fun toHumanReadableError(context: Context): HumanReadableError {
         return HumanReadableError(
             title = TITLE_ID.toResolvingString().get(context),
-            description = MESSAGE_ID.toResolvingString().get(context)
+            description = MESSAGE_ID.toResolvingString().get(context) + " ($errorCode)"
         )
```

Comment:
What happens to CLIENT_ERROR?

Reply:
Did you fix two bugs here? Wrong message and the missing append of error code?

Reply:
they are not reported , the allowlist is simply empty , so technically it will result in ALLOWLIST_MISMATCH